ui/src/templates/components/CommunityTemplateContents.tsx

```diff
@@ -0,0 +1,217 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import {connect} from 'react-redux'
+
+import {AppState, CommunityTemplate} from 'src/types'
+
+import {
+  Heading,
+  HeadingElement,
+  Page,
+  Panel,
+  ResourceList,
+  ResourceCard,
+} from '@influxdata/clockface'
+
+interface StateProps {
+  activeCommunityTemplate: CommunityTemplate
+}
+
+class CommunityTemplateContentsUnconnected extends PureComponent<StateProps> {
+  render() {
+    const {activeCommunityTemplate} = this.props
+    if (!Object.keys(activeCommunityTemplate).length) {
+      return (
+        <Page>
+          <Panel>
+            <Panel.Header>Calculating template resource needs...</Panel.Header>
+          </Panel>
+        </Page>
+      )
+    }
+
+    return (
+      <Page>
+        <ResourceList>
+          <ResourceList.Body>
+            {Array.isArray(activeCommunityTemplate.dashboards) &&
```

Comment:
here follows 150 lines of copy/pasta code that renders data about the template that will change as we decide what is valuable to display here. I'm going to hold off on codifying structure and content with abstractions and constructs until we've nailed down things a little bit more. i.e. things are going to change, this should be easy to change with it.